pkg/controller/podgroup.go

```diff
@@ -167,11 +169,21 @@ func (v *VolcanoCtrl) decoratePodTemplateSpec(pts *corev1.PodTemplateSpec, mpiJo
 	pts.Annotations[volcanov1beta1.KubeGroupNameAnnotationKey] = mpiJobName
 }
 
-func (v *VolcanoCtrl) calculatePGMinResources(_ *int32, mpiJob *kubeflow.MPIJob) *corev1.ResourceList {
-	if schedPolicy := mpiJob.Spec.RunPolicy.SchedulingPolicy; schedPolicy != nil {
+// calculatePGMinResources calculate minResources for volcano podGroup
+// The minMember is task's total MinAvailable or replicas if task's minAvailable is not set in vcJob
+// PodGroup's MinResources leaves empty now if it is not set. So we calculate the minResources among those first minMember replicas with higher priority
```

Comment:
```suggestion
// calculatePGMinResources calculates minResources for volcano podGroup.
// The minMember is task's total MinAvailable or replicas if task's minAvailable is not set in vcJob.
// PodGroup's MinResources leaves empty now if it is not set. So we calculate the minResources among those first minMember replicas with higher priority.
```

Thanks for the updates! I left a few comments about some nits. Also, Can we mention `Volcano calculates minResources iff using vcJob`?

Reply:
PodGroup's MinResources leaves empty now if it is not set. Volcano calculates minResources iff using vcJob. So we calculate the minResources among those first minMember replicas with higher priority.

like this?